### DISCUSSION TITLE
Repositorio Tarea 1

### DISCUSSION THREAD

**Discussion by bozzimuc** [IIC2133-PUC/2024-2#97] [Tarea 1] [answered]
Buenas tardes, no me ha aparecido el repositorio donde debo entregar la Tarea 1
Muchas gracias

**Comment by tomastrivino** [comment, marked as answer]
Hola, ¿creaste el repositorio con el link del generador que se encuentra en el enunciado?

**Reply by bozzimuc** [reply]
Nope. Se me pasó. Ahora sí, disculpen las molestias
